massa-consensus-exports/src/block_status.rs

```diff
@@ -58,6 +63,45 @@ impl From<&BlockStatus> for BlockStatusId {
     }
 }
 
+/// A structure defining whether we keep a full block with operations in storage
+/// or just the raw signed block without operations
+#[derive(Debug, Clone)]
+pub enum StorageOrBlock {
+    /// Keep a full storage with operations
+    Storage(Storage),
+    /// Keep only the block header and list of ops (but not the ops)
+    Block(Box<SecureShareBlock>),
+}
+
+impl StorageOrBlock {
+    /// Return a clone of the underlying block
```

Comment:
Maybe add more doc about when this is need; not obvious because of CLone & clone_block...

Reply:
done